static/js/popovers.js

```diff
@@ -510,6 +510,30 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
         e.preventDefault();
     });
+    $('body').on('click', '.view_edit_history', function (e) {
+        var msgid = $(e.currentTarget).data('msgid');
+        var row = current_msg_list.get_row(msgid);
+        var message = current_msg_list.get(rows.id(row));
+
+        popovers.hide_actions_popover();
+        $('#message-edit-history').modal("show");
+        channel.get({
+            url: "/json/message_edit_history",
+            data: {"message_id": JSON.stringify(message.id) },
+            success: function (data) {
+                // _.each(data.message_history, function (old_message) {
```

Comment:
@neerajwahi, @timabbott: any suggestion on how to convert timestamp from 1467987508 to correct time. This commented code gives incorrect date and time.